src/main/resources/openapi.json

```diff
@@ -1462,6 +1499,61 @@
                         "topic2"
                     ]
                 }
+            },
+            "SubscribedTopic": {
```

Comment:
It's more about assigned topic partitions than subscribed topics. We have the list of topics and then for each topic the assigned partitions. What do you think about AssignedTopicPartitions?